src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Attribute/index.html.twig

```diff
@@ -0,0 +1,61 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% set layoutHeaderToolbarBtn = {
+  add: {
+    href: path('admin_attributes_create', { 'attributeGroupId': attributeGroupId}),
+    desc: 'Add new value'|trans({}, 'Admin.Catalog.Feature'),
+    icon: 'add_circle_outline',
+  }
+}
+%}
+
+{% block content %}
+  {% block attributes_listing %}
+    <div class="row">
+      <div class="col">
+        {% embed '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': attributeGrid} %}
```

Comment:
Why embed and not include ?

Reply:
Include:
![Screenshot from 2019-07-15 15-38-40](https://user-images.githubusercontent.com/31609858/61216718-c358b300-a716-11e9-9eb8-95af62308e0a.png)
Embed:
![Screenshot from 2019-07-15 15-38-58](https://user-images.githubusercontent.com/31609858/61216729-c653a380-a716-11e9-8e8d-715e059f35ad.png)